mdsvex.config.js

```diff
@@ -27,7 +27,7 @@ const config = defineConfig({
 						const match = normalizeFileName(permalink) === normalizeFileName(parsedFileName);
 						return match;
 					});
-					return result !== undefined && result.length > 0 ? [toSlug(result, dir)] : '/';
+					return result !== undefined && result.length > 0 ? [toSlug(result, dir)] : ['/'];
```

Comment:
needed array to fix this

in the future should be checked if this leads to the 404 page.